File: problemas/1100/1165.dart
import 'dart:io';

void main() {
  final n = int.parse(stdin.readLineSync());

  for (var i = 0; i < n; i++) {
    int x = int.parse(stdin.readLineSync());
    bool primo = true;

    for (var j = 2; j < x; j++) {
      if (x % j == 0) {
        primo = false;
      }
    }

    stdout.writeln(x.toString() + (primo ? ' eh primo' : ' nao eh primo'));
  }
}

File: problemas/1100/1165.js
const entrada = require('fs')
  .readFileSync('/dev/stdin', 'utf8')
  .split('\n')
  .map(x => Number(x));
const n = entrada.shift();

for (let i = 0; i < n; i++) {
  const x = entrada.shift();
  let primo = true;

  for (let j = 2; j < x; j++) {
    if (x % j === 0) {
      primo = false;
    }
  }

  console.log(x + (primo ? ' eh primo' : ' nao eh primo'));
}